modules/apps/journal/journal-service/src/main/java/com/liferay/journal/service/impl/JournalFolderLocalServiceImpl.java

```diff
@@ -1402,6 +1406,8 @@ private JournalFolder _updateFolder(
 		folder.setName(name);
 		folder.setDescription(description);
 		folder.setRestrictionType(restrictionType);
+		folder.setStatusByUserId(user.getUserId());
+		folder.setStatusByUserName(user.getFullName());
```

Comment:
setStatusDate could also be added here,
ref https://github.com/liferay/liferay-portal/blob/master/modules/apps/journal/journal-service/src/main/java/com/liferay/journal/service/impl/JournalArticleLocalServiceImpl.java#L533